### ISSUE TITLE
`no-unpublished-require` not recognizing `path/index.js` in `files` with require('./path')

### ISSUE THREAD

**Issue by brettz9** [mysticatea/eslint-plugin-node#193]
In `package.json`:

```json
"files": [
    "index.js",
    "lib/index.js",
    "lib/processor.js"
  ]
```

In `index.js`, this is reported with `no-unpublished-require`:

```js
module.exports = require("./lib");
```

**Comment by mysticatea** [comment]
Thank you for your report. But I could not reproduce it. Would you provide more specific repro steps?

**Comment by brettz9** [comment]
Ugh, my apologies, was working off an older version!
